src/_data/catalog/widgets.json

```diff
@@ -586,6 +586,16 @@
     "link": "https://docs.flutter.io/flutter/cupertino/CupertinoPopupSurface-class.html",
     "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
   },
+  {
+    "name": "CupertinoScrollbar",
+    "description": "An iOS-style scrollbar that appears while scrolling lists",
```

Comment:
What about "An iOS-style scrollbar that indicates which portion of a Scrollable widget is currently visible?" That's more in line with the API docs and avoids implying that the scrollbar is just for use with lists.

Reply:
yup, your wording is better. Changed.